doc/source/whatsnew/v0.25.2.rst

```diff
@@ -61,7 +61,7 @@ Missing
 
 I/O
 ^^^
-
+- Improve infinity parsing. ``pd.read_csv()`` will now interpret ``Infinity``, ``+Infinity``, ``-Infinity`` as floating point values  (:issue:`10065`).
```

Comment:
Can you move this to 1.0.0

Reply:
Done. Also made test case more clear comparing with `float("infinity")`.
BTW not only Fortran prints infinities like that. Checkout this Java snippet
https://onlinegdb.com/r1GTsZPBB
```Java
public class Main{
     public static void main(String []args){
        System.out.println(Double.POSITIVE_INFINITY);
        System.out.println(Double.NEGATIVE_INFINITY);
     }
}
```